Project-Service-implementation-Script/Sample Templates/.~lock.ProgramTemplate.xlsx#

```diff
@@ -0,0 +1 @@
+,user3,user3-Latitude-3420,09.10.2024 12:48,file:///home/user3/.config/libreoffice/4;
```

Comment:
@MallanagoudaB what is the need fr this change?

Reply:
its a CSV change